guides/maintenance.md

```diff
@@ -0,0 +1,72 @@
+# A Maintenance Guide
+
+This manual is intended for advanced users who already able to launch an anchoring
+instance in accordance of a [newbie](newbie.md) guide.
+
+The manual describes most common procedures of the service maintenance:
+
+* [Funding of anchoring chain wallet](#Funding-of-anchoring-chain-wallet)
+* [Modification of configuration parameters](#Modification-of-configuration-parameters)
+* [Changing list of anchoring nodes](#Changing-list-of-anchoring-nodes)
+
+## Funding of anchoring chain wallet
+
+Sometimes you have to replenish the anchoring chain wallet to keep anchoring going.
```

Comment:
I'd rephrase it to something like "Anchoring process can be performed only if there is a sufficient amount of funds on the anchoring wallet. To keep anchoring working, you have to add funds to this wallet and keep the balance non-zero all the time that anchroing node is running. To increase a balance of the anchoring wallet, you should do the following:
- Get the actual anchoring address, which can be obtained...
- Save received transaction hex and wait...
- After ensuring that transaction have got enough confirmations..."